_includes/nav.html

```diff
@@ -6,9 +6,15 @@
         <a class="navbar-brand" href="{{ site.baseurl }}/">
           <img src="{{ site.baseurl }}/assets/logoOpt.png" alt="Sugar Labs logo" height="90" width="270">
         </a>
-	<div>
-	  {% include banner.html %}
```

Comment:
The reason I called from a separate html is because most of the year this will be off (commented out or deleted).

Reply:
@pikurasa i review this file please sir can you briefly discribe about this change i will do this for sure

Reply:
@pikurasa Thank you for the clarification. I understand now that banner.html is often commented out or deleted. Would you prefer that I revert this part of the change and keep banner.html as a separate file? Please let me know how you'd like me to proceed  i will surely do this.